rpi_backlight.py

```diff
@@ -144,9 +208,21 @@ def _create_argument_parser() -> argparse.ArgumentParser:
     )
     return parser
 
+def init() -> None:
+    global mode, path
+    if 'Raspberry Pi' in open('/sys/firmware/devicetree/base/model').read():
```

Comment:
It's bad practice to open a file but not close it. Suggestion:

```python
with open("/sys/firmware/devicetree/base/model") as f:
    model_information = f.read()

if "Raspberry Pi" in model_information:
    ...
elif "Tinker Board" in model_information:
    ...
else:
    raise ValueError("unsupported OS, or OS could not be detected!")
```

Don't forget to black-format the code (which changes single quotes to double quotes).

Reply:
fixed